### DISCUSSION TITLE
fooocus web paid

### DISCUSSION THREAD

**Discussion by Drake9873** [lllyasviel/Fooocus#2835] [Q&A] [answered]
hey so as I am on android  tablet running this was becoming a hassle. I'm a paid member now to use the web version. how can I import models there

**Comment by ILikeToasters** [comment]
Which platform are you paying to use Fooocus on? Fooocus itself does not have a paid version but there are services that allow you to run it online.

**Reply by Drake9873** [reply]
https://fooocus.one/apps

**Reply by ILikeToasters** [reply, marked as answer]
I think that may be some sort of scam site. It has nothing to do with this project and even on that page they are not even showing Fooocus. I could be wrong but that is the feeling I get.

**Reply by Drake9873** [reply]
damnnnn

**Reply by Drake9873** [reply]
first time happened to me, never been sca med before. seemed legit more or less all functions were carried out but seemed low end. no more fooocus for me. such a shame. back to Mage. just tried to delete my card information from the website  and they won't let me do that either.

**Reply by mashb1t** [reply]
@Drake9873 You can always fall back to using free Colab in the cloud. The only thing you need is a Google account.
Check out https://colab.research.google.com/github/lllyasviel/Fooocus/blob/main/fooocus_colab.ipynb + find more information in the readme.
